example/CMakeLists.txt

```diff
@@ -5,16 +5,8 @@
 KOKKOS_SUBPACKAGE(Example)
 
 KOKKOS_ADD_EXAMPLE_DIRECTORIES(query_device)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(fixture)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(feint)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(fenl)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(multi_fem)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(md_skeleton)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(global_2_local_ids)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(grow_array)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(sort_array)
-if(NOT Kokkos_ENABLE_CUDA)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(tutorial)
+if(NOT KOKKOS_ENABLE_CUDA)
```

Comment:
Why did you change the case?

Reply:
I thought we want to use the uppercase version everywhere for consistency.

Reply:
Nope. It is always Kokkos_ALL_CAPS for CMake variables and KOKKOS_ALL_CAPS for macros in code

Reply:
Then we are not consistent either
```
$ grep -rIn "KOKKOS_ENABLE_" cmake/ CMakeLists.txt | grep -v "SET" | grep -v "KOKKOS_ENABLE_OPTION" | grep -v "cmakedefine" | wc -l
32
```
Anyway, reverting here.